packages/prisma/src/prisma/migrations/20240816193204_element_status/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- CreateEnum
+CREATE TYPE "ElementStatus" AS ENUM ('DRAFT', 'IN_REVIEW', 'READY');
+
+-- AlterTable
+ALTER TABLE "Element" ADD COLUMN     "status" "ElementStatus" NOT NULL DEFAULT 'DRAFT';
```

Comment:
As written in the description:

_default in DB is currently set to draft -> when deploying, we should change everything already in the DB to ready. maybe this could also be done right in the migration (create the field with a default of READY and then change the default to DRAFT after within the same migration)_

however, right now I would also tend to just change it to READY always. because in the UI I also made it ready when creating an element for people who do not care. main idea with DRAFT as default was, e.g., for AI generated questions, where we want them as drafts. but we will just set draft manually there :)

Reply:
Totally missed all the description below the image in the PR description, sorry. Setting it to READY by default and only manually setting questions to the DRAFT state when generating them (or possibly also to REVIEW when importing them?) would make total sense :)